traitsui/qt4/tree_editor.py

```diff
@@ -986,8 +986,9 @@ def _on_context_menu(self, pos):
             # Use the menu specified by the node:
             group = menu.find_group(NewAction)
             if group is not None:
-                # Only set it the first time:
-                group.id = ""
+                # Reset the group for the current usage in case it is shared
+                # - the call to `node.get_add()` is potentially dynamic
+                group.clear()
```

Comment:
does the second half of the comment added for wx also apply here?

Reply:
Argh - I updated the Qt version but forgot to save and push.